src/Proxy/Proxy.csproj

```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="Linn.Common.Proxy" Version="3.0.1" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="2.2.0" />
```

Comment:
Not sure if this is just for local logging but could it just use linn.common.logging (or whatever it's called)?